react-app/src/components/GamePage.js

```diff
@@ -29,6 +29,7 @@ class Game extends Component {
     startGame: false,
     starList: [],
     trashList: [this.generateTrashState()],
+    selectedTrash: [],
```

Comment:
I thought of another benefit to using an empty array for the trash. If we made the game harder and the monster started floating we could have the monster steal and eat the trash out of 'our hands' while we're dragging it to the bin.

Reply:
I love that : )